python/src/aiconfig/editor/client/src/components/prompt/prompt_input/schema_renderer/PromptInputSchemaRenderer.tsx

```diff
@@ -95,6 +96,12 @@ export default memo(function PromptInputSchemaRenderer(props: Props) {
         value={props.input}
         label="Prompt"
         onChange={(e) => props.onChangeInput(e.target.value)}
+        onKeyDown={(event) => {
+          if (event.key === "Enter" && (event.shiftKey || event.ctrlKey)) {
```

Comment:
So does the ordering here matter? Should we instead be doing:

```
(event.shiftKey || event.ctrlKey) && event.key === "Enter"
```

so that "Enter" is the 2nd keyboard button that is pressed?

Reply:
Ordering doesn't matter -- it's one event that has both buttons pressed simultaneously (the event with just shift/ctrl key already happened and is ignored/not handled)